jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Util.java

```diff
@@ -20,29 +20,31 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
-
 package openj9.tools.attach.diagnostics.tools;
 
+import com.ibm.oti.vm.VM;
```

Comment:
I don't see this used, which means the export of com.ibm.oti.vm to jdk.jcmd isn't needed either.
Pls check the other imports, I don't think AttachNotSupportedException, VirtualMachine, AttachProvider, IOException are used either.

Reply:
Removed `import com.ibm.oti.vm.VM` which was a leftover from an earlier approach.
`AttachNotSupportedException`, `VirtualMachine`, `AttachProvider`, and `IOException` are not new additions, just re-ordering, and presently are used.